test/measures/2017/measures-data-spec.js

```diff
@@ -5,12 +5,13 @@ const fs = require('fs');
 const path = require('path');
 const parse = require('csv-parse/lib/sync');
 
-const mipsDataFormat = require('../../index.js');
-const actualAciRelation = require('../../util/measures/aci-measure-relations.json');
-const actualCpcPlusGroups = require('../../util/measures/cpc+-measure-groups.json');
-const actualMeasureSpecificationData = parse(fs.readFileSync(path.join(__dirname, '../../util/measures/measurePDF-Specification.csv'), 'utf8'));
-
 const year = 2017;
+const mipsDataFormat = require('../../../index.js');
+const actualAciRelation = require('../../../util/measures/' + year + '/aci-measure-relations.json');
+const actualCpcPlusGroups = require('../../../util/measures/' + year + '/cpc+-measure-groups.json');
+const actualMeasureSpecificationData = parse(fs.readFileSync(path.join(__dirname,
+  '../../../util/measures/' + year + '/measurePDF-Specification.csv'), 'utf8'));
```

Comment:
Weakest preference ever: if the filename of this has "2017" in it, seems fine to me to just hardcode 2017 in these strings 🤷‍♂️

Reply:
the filename doesn't, but the path does :P the only reason it's this way is because it made it easier to copy changes between the 2018 and 2017 versions of this file. which probably won't be needed anymore. but i'll leave it just in case...